src/Components/GEOSldas_GridComp/GEOSlandassim_GridComp/clsm_ensupd_upd_routines.F90

```diff
@@ -3339,17 +3339,17 @@ subroutine get_obs_pert( N_ens, N_obs, N_obs_param,                  &
     !
     ! get gridded perturbations
     
-    allocate(Obs_pert_ntrmdt(N_obs_param,pert_grid_lH%N_lon,pert_grid_lH%N_lat,N_ens))
-    allocate(Obs_pert_grid(  N_obs_param,pert_grid_lH%N_lon,pert_grid_lH%N_lat,N_ens))
+    allocate(Obs_pert_ntrmdt(pert_grid_lH%N_lon, pert_grid_lH%N_lat, N_obs_param, N_ens))
```

Comment:
Here  and in the next line, do we need to allocate with "N_obs_param" when in the call to get_pert() below we're only using "N_assim_species"?
By construction, N_assim_species <= N_obs_param. 
I'm pretty sure we can save a bit of memory.  Can you please double-check?